screw_1:
  start_frame: frame0
  target_frame: frame1
  translation: [1.0, 2.0, 3.0]
  rotation: [0.0, 0.0, 0.0, 1.0]
screw_2:
  start_frame: frame2
  target_frame: frame3
  translation: [2.0, 2.0, 2.0]
  rotation: [0.0, 0.0, 0.0, 1.0]
